source/hardware/controllers/hercules_djcontrol_inpulse_200.rst

```diff
@@ -68,8 +68,9 @@ Browser
 Control                                                          Function
 ===============================================================  ==========================================
 :hwlabel:`BROWSER` encoder (turn)                                Move up/down list.
+:hwlabel:`SHIFT` + :hwlabel:`BROWSER` encoder (turn)             Scroll up/down list.
```

Comment:
Thank you.
We have unnecessary linebreaks here because of (IMO unnecessary) "encoder".
Do you mind removing it so we only have "`BROWSER` turn" and "`BROWSER` press"?

Reply:
Sure thing, I thought the same